src/main/java/com/denizenscript/depenizen/bukkit/bridges/MythicMobsBridge.java

```diff
@@ -22,25 +24,183 @@
 import com.denizenscript.depenizen.bukkit.properties.mythicmobs.MythicMobsEntityProperties;
 import com.denizenscript.depenizen.bukkit.properties.mythicmobs.MythicMobsPlayerProperties;
 import com.denizenscript.depenizen.bukkit.utilities.mythicmobs.MythicMobsLoaders;
+import com.sucy.skill.api.skills.Skill;
 import io.lumine.mythic.api.mobs.MobManager;
 import io.lumine.mythic.api.mobs.MythicMob;
+import io.lumine.mythic.api.packs.Pack;
 import io.lumine.mythic.bukkit.BukkitAPIHelper;
 import io.lumine.mythic.bukkit.BukkitAdapter;
 import io.lumine.mythic.bukkit.MythicBukkit;
+import io.lumine.mythic.core.items.ItemExecutor;
 import io.lumine.mythic.core.items.MythicItem;
 import io.lumine.mythic.core.mobs.ActiveMob;
+import io.lumine.mythic.core.skills.SkillExecutor;
 import io.lumine.mythic.core.skills.variables.*;
 import io.lumine.mythic.core.spawning.spawners.MythicSpawner;
 import io.lumine.mythic.core.spawning.spawners.SpawnerManager;
 import org.bukkit.entity.Entity;
 import org.bukkit.inventory.ItemStack;
 
+import java.util.Collection;
 import java.util.Map;
 import java.util.Optional;
 import java.util.UUID;
 
 public class MythicMobsBridge extends Bridge {
 
+    static class MythicMobsBridgeTags extends PseudoObjectTagBase<MythicMobsBridgeTags> {
+
+        public static MythicMobsBridgeTags instance;
+
+        public MythicMobsBridgeTags() {
+            instance = this;
+            TagManager.registerStaticTagBaseHandler(MythicMobsBridgeTags.class, "mythicmobs", (t) -> instance);
+        }
+
+        @Override
+        public void register() {
+
+            // <--[tag]
+            // @attribute <mythicmobs.item_ids>
+            // @returns ListTag
+            // @plugin Depenizen, MythicMobs
+            // @description
+            // Returns a ListTag of valid MythicItem IDs. See also <@link tag mythic_item>.
+            // -->
+            tagProcessor.registerTag(ListTag.class, "item_ids", (attribute, object) -> {
+                ListTag list = new ListTag();
+                for (String item : getItemManager().getItemNames()) {
+                    list.addObject(new ElementTag(item, true));
+                }
+                return list;
```

Comment:
`ListTag`'s constructor now takes a `Collection<String>`, so should use that (and also update all other relevant places in the PR)